### ISSUE TITLE
On window resize, it drags window with it on windows os.

### ISSUE THREAD

**Issue by SahajRana** [nativeshell/nativeshell#97]
Hi @knopp, I tried this on Windows OS, and while testing I found that if you try to resize the window from the left-hand side towards the right. It starts to drag the whole window towards the right. 

The expected behavior is to keep the window in the same place and not moving it.

I have tried this on latest examples folder.

**Comment by knopp** [comment]
This is actually expected behavior when reaching minimal window size. The window doesn't resize any further, instead it keeps moving. Or is this something else?

**Comment by SahajRana** [comment]
Yaa this is exactly what is happening. But are you sure it is expected behavior, as it doesn't happen on any other kind of window & not even on the default win32 build of flutter for windows.

**Comment by knopp** [comment]
You're right. It's likely the way we're handling `WM_WINDOWPOSCHANGING`. It's certainly not regular behavior. I'll look into it.
